### ISSUE TITLE
Rotating a model before non-uniform scaling

### ISSUE THREAD

**Issue by GregValiant** [Ultimaker/Cura#8494]
This issue with non-uniform scaling is hard to explain.  I have seen this myself and others have commented on it on Reddit and the Cura forum.  It's something engineers do from time to time.
When I bring a model into Cura and rotate the model, then scale the model non-uniformly about an axis - the scaling dialog box indicates that two axis were effected, although the model appears to be correct.  If I click in the work space to de-select the model and then re-select it - the scaling dialog now indicates that the wrong axis has been scaled and the original axis that I scaled is back to 100%.  If I attempt to change back to 100% scaling on what appears to be the wrong axis - the first axis changes again to a value that is a percentage of the original scale factor.  At this point "Reset all model transformations" has no effect and the build plate has to be cleared and the STL re-opened.

If I make the transformations before rotating the model there doesn't appear to be any issues.  Rotating the model to 30, 45 etc and then making non-uniform scaling results in what to some people appears to be strange behavior as the model scales about Cura's axis, and not the models native axis.  I view that as a non-issue, just something that needs to be understood by the user.

Since some STL files open in Cura at a disadvantageous angle for slicing, then rotating the model first is a natural thing to do.  If the model is scaled uniformly, all appears to be OK.  Non-uniform scaling seems to have unintended results.
 
**Application version**
4.7beta in compatibility mode

**Platform**
Win10 home 64 bit

**Printer**
Creality Ender 3 Pro

**Reproduction steps**
1. Load an STL file
2. Rotate the model 90° about the Z axis
3. Scale the model non-uniformly 150% about the Y axis only.  Use the tab key to move out of the Y box and look at the X box. 
 It will also say 150%.  At this point the model looks correct.
4. De-select the model and then re-select it and check the scaling dialog box.  Now it says the X axis was scaled to 150% and the Y axis is now at 100% although the model still looks the same.
5. Attempting to make any further changes to the values in the Scaling dialog box results in strange behavior of the model and the numbers in the dialog box.
6. If there was a pop-up dialog explaining that non-uniform scaling should be done before rotation, that would be good.  Right now people are chasing their tails as they find that the numbers in the Scaling dialog box are moving targets.

**Screenshot(s)**
none

**Actual results**
Secondary changes to Scale in a rotated - non-uniformly scaled model results in strange skewing of the model.

**Expected results**
The model should change to the new scale factor and should do so on the correct axis.

**Project file**
none

**Log file**
none

**Additional information**
none

**Comment by nallath** [comment]
Duplicate of #4136
